server/service/src/document/patient/mod.rs

```diff
@@ -4,6 +4,12 @@ use crate::service_provider::ServiceContext;
 
 pub use self::query::*;
 mod query;
+pub use self::insert::*;
+mod insert;
+pub mod patient_document_update;
+pub mod patient_schema;
+
+pub const PATIENT_TYPE: &str = "Patient";
```

Comment:
would be good to have a comment about what this is

Reply:
[16d09fe](https://github.com/openmsupply/open-msupply/pull/61/commits/16d09fe724d36b2d745b44e1a7f5dc4433cf8e43)